apps/system-e2e/src/tests/islandis/service-portal/smoke/licenses.spec.ts

```diff
@@ -9,24 +9,34 @@ const homeUrl = `${urls.islandisBaseUrl}/minarsidur`
 test.use({ baseURL: urls.islandisBaseUrl })
 
 test.describe('MS - Skírteini', () => {
-  let context: BrowserContext
+  let contextFæreyjar: BrowserContext
+  let contextAmeríka: BrowserContext
+  const contexts: BrowserContext[] = []
 
   test.beforeAll(async ({ browser }) => {
-    context = await session({
+    contextAmeríka = await session({
```

Comment:
Is it normal to use non-ascii characters in code that is not related to user readable output?